src/common/cookie.test.tsx

```diff
@@ -27,18 +27,20 @@ describe('Cookie Utils', () => {
 
   test('setCookie sets a basic cookie', () => {
     setCookie('foo_cookie', 'bar');
-    expect(setCookieSpy).toBeCalledWith('foo_cookie=bar;path=/;secure');
+    expect(setCookieSpy).toBeCalledWith(
+      'foo_cookie=bar;path=/;SameSite=Lax;secure'
+    );
   });
 
   test('setCookie sets a basic insecure cookie', () => {
     setCookie('foo_cookie', 'bar', { secure: false });
-    expect(setCookieSpy).toBeCalledWith('foo_cookie=bar;path=/');
+    expect(setCookieSpy).toBeCalledWith('foo_cookie=bar;path=/;SameSite=Lax');
```

Comment:
Added the new default SameSite value to the expected values of all relevant tests